dictionaries/CallMap.php

```diff
@@ -7693,256 +7693,314 @@
 'MongoDB::setReadPreference' => ['bool', 'read_preference'=>'string', 'tags='=>'array'],
 'MongoDB::setSlaveOkay' => ['bool', 'ok='=>'bool'],
 'MongoDB::setWriteConcern' => ['bool', 'w'=>'mixed', 'wtimeout='=>'int'],
-'MongoDB\BSON\Binary::__construct' => ['void', 'data'=>'string', 'type'=>'int'],
-'MongoDB\BSON\Binary::__toString' => ['string'],
+'MongoDB\BSON\Binary::__construct' => ['void', 'data' => 'string', 'type' => 'int'],
 'MongoDB\BSON\Binary::getData' => ['string'],
 'MongoDB\BSON\Binary::getType' => ['int'],
-'MongoDB\BSON\binary::jsonSerialize' => ['mixed'],
-'MongoDB\BSON\binary::serialize' => ['string'],
-'MongoDB\BSON\binary::unserialize' => ['void', 'serialized'=>'string'],
-'MongoDB\BSON\binaryinterface::__toString' => ['string'],
-'MongoDB\BSON\binaryinterface::getData' => ['string'],
-'MongoDB\BSON\binaryinterface::getType' => ['int'],
-'MongoDB\BSON\dbpointer::__construct' => ['void'],
-'MongoDB\BSON\dbpointer::__toString' => ['string'],
-'MongoDB\BSON\dbpointer::jsonSerialize' => ['mixed'],
-'MongoDB\BSON\dbpointer::serialize' => ['string'],
-'MongoDB\BSON\dbpointer::unserialize' => ['void', 'serialized'=>'string'],
-'MongoDB\BSON\Decimal128::__construct' => ['void', 'value='=>'string'],
```

Comment:
In case you were wondering, the = sign after the parameter name is used to signal that the parameter is optional. Please make sure you actually wanted to remove it

Reply:
I wasn't wondering - this was quite intentional: https://github.com/mongodb/mongo-php-driver/pull/1347/files#r948261399